src/parser/jobs/mnk/Fists.js

```diff
@@ -1,22 +1,48 @@
-import _ from 'lodash'
+import Color from 'color'
 import React, {Fragment} from 'react'
+import {Pie as PieChart} from 'react-chartjs-2'
 
 import {ActionLink, StatusLink} from 'components/ui/DbLink'
-import ACTIONS from 'data/ACTIONS'
-import STATUSES from 'data/STATUSES'
+import ACTIONS, {getAction} from 'data/ACTIONS'
+import JOBS from 'data/JOBS'
+import STATUSES, {getStatus} from 'data/STATUSES'
 
 import Module from 'parser/core/Module'
 import {TieredSuggestion, SEVERITY} from 'parser/core/modules/Suggestions'
 
+// Shamelessly import Muscle Mage, you must cast Fist to continue
+import styles from '../smn/Pets.module.css'
```

Comment:
Copy the styles across or abstract them into a UI component for me. This _will_ break in less than a day (supposing I get off my ass).